CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+Installation
+============
+
+```shell
+$ git clone https://github.com/SUSE-Enceladus/suse-cloud-image-name-parser.git
+$ cd suse-cloud-image-name-parser
+
+# Activate virtual Environment then install
+# and dev dependences in editable mode
```

Comment:
Comment makes no sense.

Either:
"Activate virtual Environment and install dev dependencies in editable mode"

or:
"Activate virtual Environment then install dev dependencies in editable mode"